config/7.12.0/dbs/pim/external.yaml

```diff
@@ -5,38 +5,100 @@ processMigration:
     - metadata:
         name: "[[.ApplicationName]]-process-migration"
       data:
-        project-overrides.yml: |-
+        application.yaml: |-
+          # Default pim configuration for external databases.
+          quarkus:
```

Comment:
You shouldn't be defining the already bundled properties, only those you want to override.

Reply:
what do you mean? the content on application.yaml?
if so, see my comment below.

Reply:
same